enh(hardware-storage-dell-compellent-*) CTOR-1620 (#4770)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/hardware-storage-dell-compellent-snmp.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Connector dependencies

The following monitoring connectors will be installed when you install the **Dell Compellent SNMP** connector through the

> **Configuration > Monitoring Connector Manager** menu:

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu:
```

| Service Alias        | Service Template                                            | Service Description                          | Discovery  |
|:---------------------|:------------------------------------------------------------|:---------------------------------------------|:----------:|
| Traffic-Generic-Id   | HW-Storage-Dell-Compellent-Traffic-Generic-Id-SNMP-custom   | Check traffic of an network interface        |            |

> | Traffic-Generic-Name | HW-Storage-Dell-Compellent-Traffic-Generic-Name-SNMP-custom | Check traffic of an network interface        |            |

Reviewer: ```suggestion
| Traffic-Generic-Id   | HW-Storage-Dell-Compellent-Traffic-Generic-Id-SNMP-custom   | Check traffic of a network interface        |            |
| Traffic-Generic-Name | HW-Storage-Dell-Compellent-Traffic-Generic-Name-SNMP-custom | Check traffic of a network interface        |            |
```

### Pack

1. If the platform uses an *online* license, you can skip the package installation
instruction below as it is not required to have the connector displayed within the

> **Configuration > Monitoring Connector Manager** menu.

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu.
```

</TabItem>
</Tabs>

2. Whatever the license type (*online* or *offline*), install the **Dell Compellent SNMP** connector through

> the **Configuration > Monitoring Connector Manager** menu.

Reviewer: ```suggestion
the **Configuration > Connectors > Monitoring Connectors** menu.
```
